### PR TITLE
print optimization: rounding corners and fit margins

### PR DESCRIPTION
**Pull request by inceptionev**
rounding corners to reduce print distortions.  Fit margin added to potentiometer and lock slot pockets.

set to draft status until printed and tested

### CODE REVIEW

**Comment by inceptionev** [comment]
Print tested and successful.  Modifications reduce distortions at the corners, avoids tight fit problems around the potentiometer.

File: README.md [hunk 1]
```diff
@@ -180,6 +181,9 @@ The PAPR was designed and tested using the following setup.  You may need to mak
 - This allows the supports to come in at an angle that makes them easier to remove.
 - This is the maximum angle that still allows the supports needed to correctly print the bayonet teeth in the inlet.
 - This orientation preserves the quality of the printed text.
+- Most printing processes will require the 6-32 treads the body be chased with a 6-32 tap.  I recommend doing this step before post-curing, while the resin is still soft.
```
Comment (martukas) [inline README.md:184]: First sentence seems awkward. Also, could add the tap to the tools list:
https://www.mcmaster.com/25995A337/
or maybe this:
https://www.mcmaster.com/2529A46/
?
Also "treads"

File: README.md [hunk 2]
```diff
@@ -180,6 +181,9 @@ The PAPR was designed and tested using the following setup.  You may need to mak
...
+- Whether you chase the threads before or after post-curing, you MUST clear any uncured resin out of the thread holes before post-curing.  If the residual resin cures inside the holes, it will plug the threds.  A can of compressed gas keyboard cleaner that comes with a straw for directing the flow can be very useful here.
```
Comment (martukas) [inline README.md:185]: "threds"

File: README.md (hunk 1 above)
Comment (inceptionev) [reply README.md]: These changes are in

Comment (inceptionev) [reply README.md]: Added amazon tap tool list because full kit is about the same price as mcmaster single tap + handle

File: README.md (hunk 2 above)
Comment (inceptionev) [reply README.md]: fixed!

**Comment by inceptionev** [comment]
fixed line 5 BOM, good catch

**Comment by inceptionev** [comment]
review notes integrated, merging

**Pull request merged**